build.gradle

```diff
@@ -657,6 +657,46 @@ project(':iceberg-delta-lake') {
   }
 }
 
+project(':iceberg-bigquery') {
+  test {
+    useJUnitPlatform()
+  }
+
+  dependencies {
+    api project(':iceberg-api')
+    implementation project(':iceberg-common')
+    implementation project(':iceberg-core')
+
+    implementation project(path: ':iceberg-bundled-guava', configuration: 'shadow')
+
+    implementation platform(libs.google.libraries.bom)
+    compileOnly "com.google.cloud:google-cloud-storage"
+    implementation "com.google.cloud:google-cloud-bigquery"
+    implementation "com.google.cloud:google-cloud-core"
+
+    compileOnly libs.hadoop3.common
+    compileOnly (libs.hive2.metastore) {
```

Comment:
We can remove `libs.hive2.metastore` dependency once we inline constants of `org.apache.hadoop.hive.common.StatsSetupConst`, right? Adding this dependency just for constants looks overkill to me.

Reply:
Removed Hive dependencies.